### PR TITLE
Update the version number of sbc-common-components

### PR DESCRIPTION
**Pull request by eve-git**
*Issue #:* /bcgov/entity#6682

*Description of changes:*
Update the version number of sbc-common-components

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

**Comment by eve-git** [comment]
Passed the unit test.

**Review by severinbeauvais — APPROVED** [review]
LGTM.

Wait for second approval before merging please.

**Comment by severinbeauvais** [comment]
This PR was not supposed to be merged until after the second approval.

Therefore I have reverted this PR.

You will have to re-create this PR.

**Pull request merged**
